cost/aws/object_storage_optimization/aws_object_storage_optimization.pt

```diff
@@ -205,67 +329,28 @@ datasource "ds_aws_list_s3_objects_with_tags" do
       field "last_modified", val(iter_item, "last_modified")
       field "object_size", val(iter_item, "object_size")
       field "host", val(iter_item, "host")
+      field "modify_storage_class_to", val(iter_item, "modify_storage_class_to")
     end
   end
 end
 
 datasource "ds_aws_filtered_s3_objects" do
-  run_script $js_aws_filtered_s3_objects, $ds_aws_list_s3_objects_with_tags, $param_exclude_tags, $param_glacier_days, $param_deep_archive_days, $ds_get_caller_identity
-end
-
-
-###############################################################################
-# Scripts
-###############################################################################
-
-script "js_parse_buckets", type: "javascript" do
-  parameters "buckets"
-  result "results"
-  code <<-EOS
-    results = []
-    for ( i = 0; i < buckets.length; i++ ) {
-      if ( !buckets[i]["region"] ){
-        results.push({
-          "bucket_name": buckets[i]["bucket_name"],
-          "bucket_creation_date": buckets[i]["bucket_creation_date"],
-          "region": "us-east-1",
-          "host": "s3.amazonaws.com",
-          "tagKeyValue": ""
-        })
-      }else{
-        if(buckets[i]["region"] == "EU" ) { buckets[i]["region"] = "eu-west-1" }
-        results.push({
-          "bucket_name": buckets[i]["bucket_name"],
-          "bucket_creation_date": buckets[i]["bucket_creation_date"],
-          "region": buckets[i]["region"],
-          "host": "s3-" + buckets[i]["region"] + ".amazonaws.com",
-          "tagKeyValue": ""
-        })
-      }
-    }
-  EOS
+  run_script $js_aws_filtered_s3_objects, $ds_aws_list_s3_objects_with_tags, $param_exclude_tags, $ds_get_caller_identity
 end
 
 #Process the response data, check for the tags and generate a list of s3 objects
 script "js_aws_filtered_s3_objects", type: "javascript" do
-  parameters "ds_aws_list_s3_objects_with_tags", "param_exclude_tags", "param_glacier_days", "param_deep_archive_days", "ds_get_caller_identity"
+  parameters "ds_aws_list_s3_objects_with_tags", "param_exclude_tags", "ds_get_caller_identity"
   result "results"
   code <<-EOS
     results = [];
     var param_exclude_tags_lower=[];
     for(var i=0; i < param_exclude_tags.length; i++){
       param_exclude_tags_lower[i]=param_exclude_tags[i].toString().toLowerCase();
     }
-    var glacier_date = new Date();
-    var deep_archive_date = new Date();
-    var glacier_days_to_move = parseInt(param_glacier_days,10);
-    var deep_archive_days_to_move = parseInt(param_deep_archive_days,10);
-    glacier_date = glacier_date.setDate(glacier_date.getDate() - glacier_days_to_move);
-    deep_archive_date = deep_archive_date.setDate(deep_archive_date.getDate() - deep_archive_days_to_move);
-    glacier_date = (new Date(glacier_date)).setHours(23, 59, 59, 999);
-    deep_archive_date =(new Date(deep_archive_date)).setHours(23, 59, 59, 999);
 
     _.each(ds_aws_list_s3_objects_with_tags, function(s3Object){
+      console.log(s3Object)
```

Comment:
console.log statements are used for debugging and should be removed from production policies